### PR TITLE
Use stdin version of win-clipboard

### PR DESCRIPTION
**Pull request by floatdrop**
PR to test https://github.com/sindresorhus/win-clipboard/pull/2

Not sure, why Windows is failing completely. Maybe `copy.exe` is broken, it kinda works in virtualbox.

### CODE REVIEW

**Comment by sindresorhus** [comment]
It seems to cut off the input stream.

```js
node -e "require('.').writeSync('foo')"
```

Results in `fo` in the clipboard.

While `echo foo | copy.exe` works fine.

// @qix-

**Comment by Qix-** [comment]
I doubt it has anything to do with this, but https://github.com/sindresorhus/win-clipboard/pull/2#discussion_r116667256 needs to be addressed first and foremost.

File: lib/windows.js [hunk 1]
```diff
@@ -8,15 +8,17 @@ const winWriteBinPath = path.join(__dirname, '../fallbacks/windows/copy.exe');
 
 module.exports = {
 	copy: opts => {
-		const input = opts.input;
-		delete opts.input;
-		return execa(winWriteBinPath, [input], opts);
+		if (opts && opts.input) {
+			opts.input += '\n';
+		}
```
Comment (sindresorhus) [inline lib/windows.js]: We should fix this in `win-clipboard` instead of working around it here.

**Comment by sindresorhus** [comment]
@floatdrop Can you update the binaries from https://github.com/sindresorhus/win-clipboard/commits/master and fix the merge conflict?

**Comment by floatdrop** [comment]
@sindresorhus new binaries was merged with #23

**Pull request closed without merging**